image_proc/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ endif()
 find_package(ament_cmake_auto REQUIRED)
 ament_auto_find_build_dependencies()
 
+find_package(tracetools_image_pipeline REQUIRED)
```

Comment:
`ament_auto_find_build_dependencies()` will automatically add this if it is in `package.xml` so this line can be removed.

Reply:
Right! Thanks 621d0ffabec44e8f3b2db1812f8787c8acfbeabe.